python/hsfs/core/online_store_sql_client.py

```diff
@@ -42,11 +42,12 @@ def __init__(
         feature_store_id: id,
         skip_fg_ids: Optional[Set[int]],
         serving_keys: Optional[Set[ServingKey]] = None,
+        external: bool = True,
```

Comment:
why is the default True, should it be False so that it defaults to internal client?

Reply:
Why would we default to the internal client? Anyway the default value is not used as far as I can tell (it is resolved at the Fv level in the init), I can switch the order to make it explicit that it has to be passed.